### PR TITLE
MC-2715 Client filtering preview

### PR DESCRIPTION
**Pull request by Sodik**

### CODE REVIEW

**Review by promanenko — APPROVED** [review]

File: docs/modules/clusters/pages/client-filtering.adoc [hunk 1]
```diff
@@ -66,6 +66,60 @@ clients on the deny-list shut down.
...
+|Version of the client codebase.
+
+|===
+
+[[cleint-filtering-deploy-confirmation]]
+== Deploy confirmation
+
+image:ROOT:ClientFilteringDeployConfirmation.png[Client Filtering Deploy Comfirmation]
+
+The deployment of filter lists must be confirmed by clicking the *Confirm Deploy* button in a modal.
```
Comment (rebekah-lawrence) [inline docs/modules/clusters/pages/client-filtering.adoc:117]:
```suggestion
The deployment of filter lists must be confirmed by clicking the *Confirm Deploy* button.
```

File: docs/modules/clusters/pages/client-filtering.adoc [hunk 2]
```diff
@@ -66,6 +66,60 @@ clients on the deny-list shut down.
...
+This modal provides a summary of clients connectivity after the current filter lists have been applied.
```
Comment (rebekah-lawrence) [inline docs/modules/clusters/pages/client-filtering.adoc:118]:
```suggestion
A summary of client connectivity is shown after the current filter lists have been applied.
```

File: docs/modules/clusters/pages/client-filtering.adoc [hunk 3]
```diff
@@ -66,6 +66,60 @@ clients on the deny-list shut down.
...
+<<client-filtering-preview, Preview>> can be enabled by clicking a toggle control.
+
+IMPORTANT: If current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) will be shown on the top of a modal.
```
Comment (rebekah-lawrence) [inline docs/modules/clusters/pages/client-filtering.adoc:121]:
```suggestion
IMPORTANT: If the current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) will be shown on the top of a modal.
```

**Review by rebekah-lawrence — CHANGES_REQUESTED** [review]
A few minor suggestions

File: docs/modules/ROOT/images/ClientFilteringDeployConfirmation.png
Comment (anatoly-hazelcast) [inline docs/modules/ROOT/images/ClientFilteringDeployConfirmation.png:1]: I'm afraid Connected after and Remain connected is effectively the same as we can't identify potential future clients.
https://hazelcast.atlassian.net/browse/MC-2829

**Review by anatoly-hazelcast — APPROVED** [review]

File: docs/modules/clusters/pages/client-filtering.adoc [hunk 4]
```diff
@@ -66,6 +66,60 @@ clients on the deny-list shut down.
...
+[[cleint-filtering-deploy-confirmation]]
+== Deploy confirmation
+
+image:ROOT:ClientFilteringDeployConfirmation.png[Client Filtering Deploy Comfirmation]
+
+The deployment of filter lists must be confirmed by clicking the *Confirm Deploy* button.
+A summary of client connectivity is shown after the current filter lists have been applied.
+<<client-filtering-preview, Preview>> can be enabled by clicking a toggle control.
+
+IMPORTANT: If the current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) will be shown on the top of a modal.
```
Comment (rebekah-lawrence) [inline docs/modules/clusters/pages/client-filtering.adoc:121]:
```suggestion
IMPORTANT: If the current filter lists disconnect all clients, the warning (*All clients will be blocked if you apply current client filtering configuration*) is displayed on the top of a modal.
```

**Review by rebekah-lawrence — APPROVED** [review]
One teeny tiny suggestion.

Otherwise LGTM, so approving in advance.

**Pull request merged**